mmv1/third_party/terraform/services/identityplatform/resource_identity_platform_config_test.go

```diff
@@ -122,6 +129,13 @@ resource "google_identity_platform_config" "basic" {
         }
     }
   }
+  sms_region_config {
+    allowlist_only {
+      allowed_regions = [
```

Comment:
As the `allowed_regions` is a list, can you please add more items to the list?

Reply:
Done!